python/pyspark/sql/tests/test_functions.py

```diff
@@ -995,62 +995,6 @@ def test_make_timestamp_ntz(self):
         with self.assertRaises(Exception):
             F.make_timestamp_ntz(date=df_dt.date)
 
-        # Test 17: Invalid data types - should raise exception for invalid string to int cast
```

Comment:
How about changing it to

```python
with self.sql_conf({"spark.sql.ansi.enabled": True}):
            # Test 17: Invalid data types - should raise exception for invalid string to int cast
            with self.assertRaises(Exception):
                self.spark.range(1).select(
                    F.make_timestamp_ntz(
                        F.lit("invalid"), F.lit(5), F.lit(22), F.lit(10), F.lit(30), F.lit(0)
                    )
                ).collect()
....
```

Reply:
I feel we don't need to check this from pyspark's, it is guaranteed by spark sql.